src/Psalm/Internal/PhpVisitor/ReflectorVisitor.php

```diff
@@ -2122,8 +2127,14 @@ private function registerFunctionLike(PhpParser\Node\FunctionLike $stmt, $fake_m
         $doc_comment = $stmt->getDocComment();
 
 
-        if ($class_storage && ! $class_storage->is_trait) {
-            $storage->internal = $class_storage->internal;
+        if ($class_storage &&
+                !$class_storage->is_trait &&
```

Comment:
Small style thing: I prefer `&&` at the start of the line, not the end, as it makes it _slightly_ easier to scan

Reply:
2df047136